cowprotocol/cowswap
# Progress bar iteration 2

# Summary

Close https://github.com/cowprotocol/cowswap/issues/615, #793 

![image](https://user-images.githubusercontent.com/622217/176168924-b520f63f-8d41-4f24-a3ee-2713d48a0290.png)
![image](https://user-images.githubusercontent.com/622217/176169030-da3bc4e2-3c98-454f-9674-54ab4edb10b2.png)

[UX Progressbar feedback](https://docs.google.com/document/d/1ZIfWZMu65v-WWCiqeDoLXO272l385Y4_kGS9smP1eaQ/edit)

# To Test

1.  Make a trade
2.  Observe the progress bar
